blocks/layout-grid/src/grid-column/edit.native.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { InnerBlocks } from '@wordpress/block-editor';
+import { Component } from '@wordpress/element';
+
+class Edit extends Component {
```

Comment:
I don't have a strong feeling about this I tried to follow the convention that already exists in the other files. 
I think this can be addressed in subsequent PRs. Since I am expecting this to change quite a bit :)